### PR TITLE
Provide access to raw tag value.

### PR DESCRIPTION
**Pull request by Naarakah**
An implementation for https://github.com/felixc/rexiv2/issues/15.
I'm still new to rust so let me know if there is any issue with my code.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -30,6 +30,10 @@ include = [
 gexiv2-sys = "1.0"
 libc = "0.2"
 num-rational = { version = "0.2", default-features = false }
+glib-sys = { version = "*", optional = true }
```
Comment (felixc) [inline Cargo.toml]: Is it safe to depend on `version = "*"`, or should we limit it to something known to be supported?

**Comment by felixc** [comment]
Hey, need any help to get this landed? Do the steps in https://github.com/felixc/rexiv2/pull/32#pullrequestreview-261602272 make sense? Thanks again!

**Comment by Naarakah** [comment]
Hey, sorry for the delay
I think I made all the required changes.

**Comment by felixc** [comment]
Thank you again for this improvement! I've merged it in with https://github.com/felixc/rexiv2/commit/5a90ebca7a0becbdafad923069a57febf59968ae. GitHub didn't recognize it because I had to tweak the commit in order to include a fix from https://github.com/felixc/gexiv2-sys/commit/11e04f4cf073de591e99c65cee1896b99a928962 because I'd broken the `gexiv2-sys` dependency in the meantime. Marking it as "closed" but it was in fact merged.

**Pull request closed without merging**